homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+Linus, 2+2 is 4, not 5, PUT ATTENTION STOP PLAYING POKEMON GO ON CLASSES!
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White, Linus are you daltonic?
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240, I need to accept I didn't even knew this.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language (HTML) (I didn't even knew this)
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
```

Comment:
You can definetely make your website look pretty with CSS!
There is also another important aspect of it, which explains why we have some people specialized in graphic / web design and some specialized in code / content creation.
Can you tell us what that important aspect is? :)